.idea/compiler.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
+    <bytecodeTargetLevel>
+      <module name="ide-metrics-common_main" target="1.8" />
+      <module name="ide-metrics-common_test" target="1.8" />
+    </bytecodeTargetLevel>
+  </component>
+</project>
```

Comment:
should we fix those end of lines so Github won't complain?

Reply:
done